### PR TITLE
Add graphic list view

### LINKED ISSUE #654: Layer list like photoshop

### PR DESCRIPTION
**Pull request by bigfoodK**
Animation

- Show what graphics are in the cut
- Change graphic order by dragging graphic list item
  - Show cursor where graphic to be placed
- Select graphic on wysiwyg editor with graphic list

Closes #654

### CODE REVIEW

**Comment by namse** [comment]
Did you handled ctrl+z and ctrl+y or ctrl+shift+z for un/redo?

**Comment by bigfoodK** [comment]
> Did you handled ctrl+z and ctrl+y or ctrl+shift+z for un/redo? Please put that function.

I didn't know that. But `un/redo` seems to be working.

**Comment by namse** [comment]
Please check that

**Comment by bigfoodK** [comment]
Yeah, It definitely works.
It will be handled [here](https://github.com/NamseEnt/namseent/blob/73a7086d14dddb5a5835ea0c8fc72d084ac34dcc/luda-editor/client/src/pages/sequence_edit_page/components/cut_editor/background_with_event.rs#L104-L107)

**Review by namse — CHANGES_REQUESTED** [review]

File: luda-editor/client/src/pages/sequence_edit_page/components/graphic_list_view/graphic_list_item.rs [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+                                size: 12.int_px(),
+                                name: "NotoSansKR-Regular".to_string(),
+                            },
+                            style: TextStyle {
+                                color: stroke_color,
+                                ..Default::default()
+                            },
+                            max_width: None,
+                        }));
+                    })(wh, ctx);
```
[the remark this answers is not in the document]
Comment (bigfoodK) [reply luda-editor/client/src/pages/sequence_edit_page/components/graphic_list_view/graphic_list_item.rs:64]: Ah, you mean `namui_prebuilt::typography::body::left`. right?
I will do this now

File: luda-editor/client/src/pages/sequence_edit_page/components/side_bar/mod.rs [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        } = self;
+
+        ctx.compose(|ctx| {
+            table::hooks::vertical([
+                table::hooks::ratio(1, |wh, ctx| {
+                    ctx.add(memo_list_view::MemoListView {
+                        wh,
+                        memos,
+                        user_id,
+                        on_event: Box::new(|event| on_event(Event::MemoListView(event))),
```
[the remark this answers is not in the document]
Comment (bigfoodK) [reply luda-editor/client/src/pages/sequence_edit_page/components/side_bar/mod.rs]: fixed

**Review by namse — APPROVED** [review]
LGTM

File: luda-editor/client/src/pages/sequence_edit_page/components/graphic_list_view/mod.rs [hunk 3]
```diff
@@ -0,0 +1,222 @@
...
+                                                        }
+                                                    }
+                                                })
+                                                .with_mouse_cursor(MouseCursor::Pointer),
+                                        ),
+                                    };
+                                })(wh, ctx);
+                            })
+                        }))(wh, ctx);
+                    },
```
[the remark this answers is not in the document]
Comment (bigfoodK) [reply luda-editor/client/src/pages/sequence_edit_page/components/graphic_list_view/mod.rs]: Not the best but reduced nest

**Pull request merged**